test/core/ExamplesUnitTests/TribitsExampleProject2_Tests.cmake

```diff
@@ -1211,6 +1211,21 @@ function(TribitsExampleProject2_External_RawPackage1_then_Package_by_Package
       "Error, findingTplsMethod='${findingTplsMethod}' is invalid!")
   endif()
 
+  if (package1UseTribitsTestFunctions STREQUAL "PACKAGE1_USE_TRIBITS_TEST_FUNCTIONS")
+    set(package1UseTribitsTestFunctionsArgs
+      "-D Package1_USE_TRIBITS_TEST_FUNCTIONS=TRUE" )
```

Comment:
Again, did you mean to put a space here? I figured this would get passed to the test as a single argument with a space in it rather than two arguments.

Reply:
I don't think you need the space here.  I will try changing it.  I just get so confused all the time about how CMake deals with arguments and whitespace.

Reply:
Resolved in new commit ca2399ac949a1af5e8b50a2556acdcd4bed20c67.